### ISSUE TITLE
Feature: Grid displayed in overlay should represent the configured shape of the virtual grid

### ISSUE THREAD

**Issue by seraasch** [cspotcode/virtual-desktop-grid-switcher#14]
Would be cool if the "grid" represented the shape in the configuration also... E.g. I use a 6x1 grid that shows up as a 3x3 in the overlay

**Comment by cspotcode** [comment]
Definitely.  I think I'll have to replace those icons with a manually-rendered grid using AHK's GUI commands.

How do you think I should update the tray icon for strange grid dimensions?  Right now those icons work well for the 3x3 grid but I'm not sure other dimensions can be represented in a tiny icon.  Should I just show a number?

**Comment by seraasch** [comment]
For the tray icon, I don't know that there _is_ a good solution. What I've seen before is simply pacing the digit(s) representing the desktop number.
